sirepo/package_data/static/json/schema-common.json

```diff
@@ -147,8 +147,9 @@
             "maxSimCopies": 100,
             "oneDayMillis": 86400000,
             "plans": {
```

Comment:
I think this is to terse. This is "paymentPlans".

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/110ea9c3b63ca85561bac84b1dd582a383edab18